charts/enforcer-k8s/templates/deployment.yaml

```diff
@@ -32,9 +32,17 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" ($.Chart.AppVersion) )  }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
+            - "--port={{ .Values.service.port }}"
+            - "--debug={{ .Values.debug }}"
             - "--tls-secret={{ .Release.Namespace }}/{{ .Values.tlsSecretName }}"
             - "--policy-id={{ .Values.policyId }}"
             - "--rode-host={{ .Values.rode.host }}"
+            - "--rode-insecure={{ .Values.rode.insecure }}"
+            - "--registry-insecure-skip-verify={{ .Values.registryInsecureSkipVerify }}"
+            - "--k8s-in-cluster={{ .Values.kubernetes.inCluster }}"
```

Comment:
I almost didn't expose this last two, since at the moment they're mainly aimed at local dev. That said I could see a use case if there's a central mgmt/services cluster that runs the enforcer (with a separate validating webhook config defined in the app clusters).

Reply:
yeah I agree we should expose it.  but we should add a guard rail in the chart so we don't ever accidentally use both `--k8s-in-cluster` and `--k8s-config-file` at the same time

Reply:
Sounds reasonable, I went ahead and added that guard.

Reply:
this approach still looks like it would set both flags in some instances, right?  or is that okay?

Reply:
I think we have to still allow for setting `--k8s-in-cluster=false` when `--k8s-config-file` has a value, otherwise it would get the [default](https://github.com/rode/enforcer-k8s/blob/502f4f42a522e846a318ea90f97c3642f0531ec5/config/config.go#L70) and [ignore](https://github.com/rode/enforcer-k8s/blob/502f4f42a522e846a318ea90f97c3642f0531ec5/main.go#L107) the config file.